source/vstd/hash_set.rs

```diff
@@ -85,6 +104,8 @@ impl<Key> HashSetWithView<Key> where Key: View + Eq + Hash {
         self.m.insert(k)
     }
 
+    /// Removes the given value from the set. Returns true if the value was previously in the set, false otherwise.
```

Comment:
No, it returns true if the value *wasn't* previously in the set, false otherwise.